sale_order_product_recommendation/i18n/pt_BR.po

```diff
@@ -4,17 +4,16 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 12.0\n"
+"Project-Id-Version: Odoo Server 15.0\n"
```

Comment:
Why there are modifications in this file in the migration to 15.0?

Reply:
But why exporting this specific language and not all, or the Spanish one that is the one that we are affected?